lib/literature/live/publication_live.ex

```diff
@@ -36,7 +36,7 @@ defmodule Literature.PublicationLive do
           publication={@publication}
           action={@live_action}
           return_to={@return_to}
-          available_languages={[{"", ""}] ++ Literature.Language.available_languages()}
+          available_languages={Literature.Language.available_languages()}
```

Comment:
Why did we remove the appended blank options?

Reply:
I added prompt in `publication_form_componenet.ex`